resident-services/support-files/applicant_id.csv

```diff
@@ -0,0 +1,767 @@
+applicantId
+002
```

Comment:
I believe this is applicantTypeId, i think we have to rename file, header, variable in the script etc..

Reply:
Done